main.tf

```diff
@@ -89,7 +91,6 @@ module "google_mysql_db" {
     retained_backups               = null
     retention_unit                 = null
   }
-
```

Comment:
Keep the new line to show that below stuff are different

Reply:
done